.gitignore

```diff
@@ -1,12 +1,13 @@
+.terraform/plugins
 .vscode
 .venv
 __pycache__
-data_ingestion/__pycache__
-.terraform/plugins
 apply_terraform
-jenniebrown.tfvars
+jenniebrown*
```

Comment:
Btw, you can have a personal git exclude rule, which I think is preferable for things like this. https://www.atlassian.com/git/tutorials/saving-changes/gitignore#personal-git-ignore-rules

Reply:
Cool, I didn't know about that!